homeassistant/components/deconz/switch.py

```diff
@@ -62,16 +74,17 @@ class DeconzPowerPlug(DeconzDevice, SwitchEntity):
     """Representation of a deCONZ power plug."""
 
     TYPE = DOMAIN
+    _device: Light
 
     @property
-    def is_on(self):
+    def is_on(self) -> bool:
         """Return true if switch is on."""
-        return self._device.state
+        return bool(self._device.state)
```

Comment:
```suggestion
        return self._device.state  # type: ignore[no-any-return]
```
When the lib exports types mypy will warn you 
> unused "type: ignore" comment
the `bool()` cast will not generate a warning - so if it is not needed (e.g. to cast `None`) I'd not use it.